en/docs/integration-guides/usecases/datamapper/read-csv-file-and-transform-to-xml-file.md

```diff
@@ -209,44 +209,48 @@ The generated structure will appear as:
         └─ price: decimal
    ```
 
-You now have an **record type** that defines the exact structure of your output xml file.  
-This type will act as the **target structure** in the Data Mapper,  
+You now have an record type that defines the exact structure of your output xml file.  
+This type will act as the target structure in the Data Mapper,  
```

Comment:
The above final step is not numbered. It should be `4. Click **Import** to load the sample ...`

<img width="877" height="462" alt="image" src="https://github.com/user-attachments/assets/fc16930c-a9b5-46d6-923b-5e642b2c5896" />